src/WeekCalendar/CalendarCells.tsx

```diff
@@ -0,0 +1,12 @@
+import styles from './weekCalendar.module.css';
+import { DAYS_IN_WEEK, CELLS_IN_COLUMN } from '../constants';
+import { createArray } from '../utils';
+const CalendarCells = () => (
+  <div className={styles.calendarCells}>
+    {createArray(DAYS_IN_WEEK * CELLS_IN_COLUMN).map((_, i) => {
+      return <div className={styles.cell} key={i}></div>;
```

Comment:
it is not the best idea to use index for key, read:
pitfall section: https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key